client/components/typography/typography.css

```diff
@@ -0,0 +1,48 @@
+@import url("https://fonts.googleapis.com/css?family=IBM Plex Sans");
+
+body {
+    font-family: "IBM Plex Sans";
+    font-style: normal;
+}
+
+.heading-h2 {
+    font-weight: 600;
+    font-size: 34px;
+    line-height: 44px;
+    letter-spacing: 0em;  
+}
+
+.heading-h3 {
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 34px;
+    letter-spacing: 0em;
+}
+
+.heading-h5 {
+    font-size: 18px;
+    font-weight: 600;
+    line-height: 28px;
+    letter-spacing: 0em;
+}
+
+.paragraph-default {
+    font-size: 18px;
+    font-weight: 300;
+    line-height: 28px;
+    letter-spacing: 0em;
+}
+
+.other-eyebrows {
```

Comment:
I named classes according to Figma names :) 
![image](https://user-images.githubusercontent.com/10188652/196390990-9a310d21-6f09-4f4b-ab87-34ba8ece8123.png)

I thought all participants saw Figma so it would be easy to find out neсessary class if it named like in Figma :)
But you are right. I'll change class name

Reply:
changed